packages/otel/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@hono/otel",
+  "version": "0.1.0",
```

Comment:
The version should be `0.0.0`. The changeset will update the version automatically.